docs/using/alerting.md

```diff
@@ -2,7 +2,7 @@
 
 *Integrated Alerting* lets you know when certain system events occur.
 
-> <b style="color:goldenrod">Caution</b> Integrated Alerting is a [technical preview](../../../details/glossary.md#technical-preview) and is subject to change.
+> <b style="color:goldenrod">Caution</b> Integrated Alerting is a [technical preview](../details/glossary.md#technical-preview) and is subject to change.
```

Comment:
is this for release notes?

Reply:
No, it was a change in my main that got caught up. You can ignore it (just a cosmetic item).